src/os/macos.rs

```diff
@@ -14,22 +14,26 @@ struct RawConnection {
     process_name: String,
 }
 
-pub(crate) fn get_open_sockets() -> HashMap<Connection, String> {
+#[allow(clippy::needless_return)]
```

Comment:
Why?

Reply:
This is to surprise compiler warning about using explicit returns. I removed the returns and this #[allow]